float64 offset #s10_timestamp - mcu_timestamp
---
string response
